packages/harmony/tsconfig.json

```diff
@@ -4,22 +4,24 @@
     "lib": ["ES2020", "dom"],
     "module": "esnext",
     "checkJs": false,
+    "plugins": [{ "name": "typescript-plugin-css-modules" }],
     "allowJs": false,
     "jsx": "react-jsx",
     "resolveJsonModule": true,
     "moduleResolution": "node",
     "typeRoots": [
```

Comment:
wondering why we need type-roots for harmony but not stems?

Reply:
not needed, but nice to have for autocompletion of the css module classnames

Reply:
dope this is so great